src/index.js

```diff
@@ -73,10 +74,22 @@ export class WT {
   }
 
   initialize(payload) {
+    this.elapsedTime = 0;
     this.wtConfig = resolveMethod(payload, this.wtConfig, this);
     if (this.wtConfig.cookies) { this.getVisitorToken(); }
   }
 
+  pageview(...payload) {
+    const t = this;
+    t.handleEvent('pageview', ...payload);
+    if (t.wtConfig.pingInterval) {
```

Comment:
two options

1) move the set interval inside initialize
2) keep it in page view but do a clear interval

option 2 is a bit more idea

Reply:
also, let's not pass elapsed time --  we already get the ts, and we can save this row, and in query, duration can be expressed as pageviews.ts - pageviews.last_ping (last_ping is set via pingevent.ts)

Reply:
also, we need to add UUID to this PR, should reset on pageview events.